### PR TITLE
PMM-12641 Cleanup server build scripts

### PR DESCRIPTION
**Pull request by ademidoff**
PMM-12641

Link to the Feature Build: SUBMODULES-3720

If this PR adds or removes or alters one or more API endpoints, please review and add or update the relevant [API documents](https://github.com/percona/pmm/tree/main/docs/api) as well:

- [ ] API Docs updated

If this PR is related to some other PRs in this or other repositories, please provide links to those PRs:

- Links to related pull requests (optional).

### CODE REVIEW

File: build/docker/server/Dockerfile.el9 [hunk 1]
```diff
@@ -38,10 +30,21 @@ COPY pmm-client.tar.gz /tmp/
 RUN install -T -p -m 644 /opt/ansible/ansible.cfg /etc/ansible/ansible.cfg && \
     ansible-playbook -vvv -i 'localhost,' -c local /opt/ansible/pmm-docker/main.yml -e "pmm_running_in_docker=true" && \
     ansible-playbook -vvv -i 'localhost,' -c local /opt/ansible/pmm-docker/update.yml -e "pmm_running_in_docker=true" && \
-    ansible-playbook -vvv -i 'localhost,' -c local /opt/ansible/pmm/post-build-actions.yml -e "pmm_running_in_docker=true"
+    ansible-playbook -vvv -i 'localhost,' -c local /opt/ansible/pmm/post-build-actions.yml -e "pmm_running_in_docker=true" && \
```
Comment (ademidoff) [inline build/docker/server/Dockerfile.el9:33]: @talhabinrizwan FYI: I'm removing those `pmm_running_in_docker` vars as they don't seem to be used in v3.

**Review by idoqo — APPROVED** [review]

File: build/docker/server/Dockerfile.el9 [hunk 2]
```diff
@@ -36,12 +28,23 @@ COPY gitCommit /tmp/gitCommit
 COPY pmm-client.tar.gz /tmp/
 
 RUN install -T -p -m 644 /opt/ansible/ansible.cfg /etc/ansible/ansible.cfg && \
-    ansible-playbook -vvv -i 'localhost,' -c local /opt/ansible/pmm-docker/main.yml -e "pmm_running_in_docker=true" && \
-    ansible-playbook -vvv -i 'localhost,' -c local /opt/ansible/pmm-docker/update.yml -e "pmm_running_in_docker=true" && \
-    ansible-playbook -vvv -i 'localhost,' -c local /opt/ansible/pmm/post-build-actions.yml -e "pmm_running_in_docker=true"
```
Comment (BupycHuk) [inline build/docker/server/Dockerfile.el9:41]: is pmm_running_in_docker used anywhere?

**Review by BupycHuk — APPROVED** [review]

File: build/docker/server/Dockerfile.el9 (hunk 1 above)
Comment (ademidoff) [reply build/docker/server/Dockerfile.el9]: No, it's not used in v3 :)

File: build/docker/server/Dockerfile.el9 (hunk 2 above)
Comment (ademidoff) [reply build/docker/server/Dockerfile.el9:41]: No, it's not used in v3 :)

**Review by talhabinrizwan — APPROVED** [review]

**Pull request merged**